sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/src/Models/TimeSeries2Id.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Iot.TimeSeriesInsights
+{
+    /// <summary>
+    /// A single Time Series Id value that is an array of 2 string values that uniquely identifies a Time Series instance.
+    /// </summary>
+    /// <remarks>
+    /// At the moment, Time Series Id service only supports string types to represent a TSI Id. In order to reduce the confusion on the user,
+    /// this class has been created to extend the more generic class <see cref="TimeSeriesId{T1, T2}"/>.
+    /// </remarks>
+    public class TimeSeries2Id : TimeSeriesId<string, string>
+    {
+        /// <summary>
+        /// Creates a new Time Series Id with 2 string properties.
```

Comment:
Creates a new instance of class see cref ...

Reply:
class has been removed as we're trying a new design for TimeSeriesId :)